docs/developers/guides/token.md

```diff
@@ -0,0 +1,1320 @@
+
+    [File]: assembly/__tests__/Token.spec.ts
+  [Groups]: 2 pass, 2 total
+  [Result]: ✔ PASS
+[Snapshot]: 0 total, 0 added, 0 removed, 0 different
+ [Summary]: 13 pass,  0 fail, 13 total
+    [Time]: 178.067ms
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  [Result]: ✔ PASS
+   [Files]: 1 total
+  [Groups]: 2 count, 2 pass
+   [Tests]: 13 pass, 0 fail, 13 total
+    [Time]: 5465.975ms
+┌───────────────────┬───────┬───────┬──────┬──────┬───────────┐
+│ File              │ Total │ Block │ Func │ Expr │ Uncovered │
+├───────────────────┼───────┼───────┼──────┼──────┼───────────┤
+│ assembly/Token.ts │ 100%  │ 100%  │ 100% │ 100% │           │
+├───────────────────┼───────┼───────┼──────┼──────┼───────────┤
+│ total             │ 100%  │ 100%  │ 100% │ 100% │           │
+└───────────────────┴───────┴───────┴──────┴──────┴───────────┘
+
+
+```
+
+---
+## Compiling the contract
+Once you've made all the necessary modifications and your tests are passing, you're ready to build the release version of your contract. Let's run the build script for release.
+
+```sh
+koinos-sdk-as-cli build-all release 0 token.proto
+```
+
+```{ .txt .no-copy }
+Generating ABI file...
+ yarn protoc --plugin=protoc-gen-abi=./node_modules/.bin/koinos-abi-proto-gen --abi_out=abi/ assembly/proto/token.proto
+yarn run v1.22.19
+warning ../../../../../package.json: No license field
+$ /Users/ron/devstuff/projects/tmp/token-tutorial/token/node_modules/.bin/protoc --plugin=protoc-gen-abi=./node_modules/.bin/koinos-abi-proto-gen --abi_out=abi/ assembly/proto/token.proto
+installing estraverse@^5.1.0
+Done in 2.26s.
+Generating proto files...
+yarn protoc --plugin=protoc-gen-as=./node_modules/.bin/as-proto-gen --as_out=. assembly/proto/*.proto
+yarn run v1.22.19
+warning ../../../../../package.json: No license field
+$ /Users/ron/devstuff/projects/tmp/token-tutorial/token/node_modules/.bin/protoc --plugin=protoc-gen-as=./node_modules/.bin/as-proto-gen --as_out=. assembly/proto/token.proto
+Done in 0.46s.
+Generating boilerplate.ts and index.ts files...
+yarn protoc --plugin=protoc-gen-as=./node_modules/.bin/koinos-as-gen --as_out=assembly/ assembly/proto/token.proto
+yarn run v1.22.19
+warning ../../../../../package.json: No license field
+$ /Users/ron/devstuff/projects/tmp/token-tutorial/token/node_modules/.bin/protoc --plugin=protoc-gen-as=./node_modules/.bin/koinos-as-gen --as_out=assembly/ assembly/proto/token.proto
+Done in 0.38s.
+Compiling index.ts...
+node ./node_modules/assemblyscript/bin/asc assembly/index.ts --target release --use abort= --use BUILD_FOR_TESTING=0 --disable sign-extension --config asconfig.json
+```
+
+After the build completes, locate your `.wasm` and `.abi` files:
+
+- `.wasm` file: `./build/release/contract.wasm`
+- `.abi` file: `./abi/token.abi`
+
+With these files, let's generate a Koinos wallet with `koinos-cli` and deploy your contract.
+
+---
+
+## Deploying the contract
```

Comment:
This whole section should link to the existing `Deploy a contract` section instead of repeating the information.